cn_indian_payroll/cn_indian_payroll/page/payroll_configuratio/payroll_configuratio.js

```
@@ -735,590 +32,392 @@ frappe.pages['payroll-configuratio'].on_page_load = function(wrapper) {
 
-											msgprint("Component Already Added")
 
-										}
+										d.hide();
 									}
-								})
+								});
+							
+								d.show();
+							
+								let tableField = d.fields_dict.salary_components_table;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'tableField' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)